WhatsNew/Items/WhatsNew.swift

```diff
@@ -9,31 +9,38 @@ import Foundation
 
 public struct WhatsNew {
     static let bundle = Bundle(for: WhatsNewViewController.self)
-    static let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
-    static let userDefaultsKey = "LatestAppVersionPresented"
+    public static let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String
```

Comment:
Can you rename this variable to `currentVersion` and move it before all the `internal` declarations so it's easier to see it's `public`?

Reply:
of course!